.github/workflows/packages-release.yaml

```diff
@@ -0,0 +1,34 @@
+# This workflow is intended to be run manually when cutting an official release.
```

Comment:
Is there a reason not to do this in the packages workflow? ie is there a cost to generating the SBOM for every packages run?

Reply:
I was originally thinking we'd want to restrict permission to run the SBOM job to only Microsoft devs. However, that's probably not necessary and more work would have to be done to accomplish that goal, so I'll just move this to packages.yaml.